BitwardenShared/Core/Platform/Models/Enum/FeatureFlag.swift

```diff
@@ -29,6 +29,11 @@ enum FeatureFlag: String, CaseIterable, Codable {
     /// An SDK flag that enables individual cipher encryption.
     case enableCipherKeyEncryption
 
+    /// A flag to ignore the environment check for the two-factor authentication
+    /// notice. If this is on, then it will display even on self-hosted servers,
+    /// which means it's easier to dev/QA the feature.
+    case ignoreEnvironmentCheck = "ignore-environment-check"
```

Comment:
⛏️ Could some more context be added to the name of the flag? e.g. `ignore2faNoticeEnvironmentCheck` or something that is more descriptive? I feel the flag to be too broad as which environment check it should ignore (if one were to rely on the flag name and not on the well detailed comment)